tox.ini

```diff
@@ -31,8 +31,8 @@ commands   = sphinx-build -W -b html -d {envtmpdir}/doctrees .  {envtmpdir}/html
 basepython = python3
 skip_install = true
 deps =
-    black
-    isort
+    black~=19.10b0
```

Comment:
I'm iffy on this pin; can you refresh my memory as to what `~=` means? I just re-read PEP-508 and I couldn't find *anything*

Reply:
It's defined in [PEP-440](https://www.python.org/dev/peps/pep-0440/) - it's a [compatible release](https://www.python.org/dev/peps/pep-0440/#compatible-release). 

Short version as I understand it - in theory, any new version which should be compatible will match, but it won't move up a major version. 19.0, 19.1 and so on would match, but 20.0 will not.